packages/vitest/src/node/index.ts

```diff
@@ -403,12 +403,8 @@ export async function createVitest(options: UserConfig, viteOverrides: ViteUserC
   const server = await createServer(mergeConfig(config, viteOverrides))
   await server.pluginContainer.buildStart({})
 
-  if (options.api === true)
-    options.api = defaultPort
-  if (options.open && typeof options.api !== 'number')
-    options.api = defaultPort
-  if (typeof options.api === 'number')
-    await server.listen(options.api)
+  if (options.api)
```

Comment:
should we protect here using `if (typeof options.api === 'object')`? rn only called from the cli (from IDE usages)

Reply:
I'm moveing the `api` configuration to `src/node/index.ts`...

Reply:
I'm moving the configuration before creating the server, and so there is no need to do it twice.